src/Chapter1.hs

```diff
@@ -620,7 +630,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = 
+  let absn = abs n
+      last2 = div (mod absn 100) 10
+      last1 = mod absn 10
```

Comment:
Perfect solution! 👍🏼 

You may notice, that `last1` looks the same as `lastDigit` from the previous task 😉

Reply:
I will make this change in my next commit. Thank you for the suggestion